Client/Application/WebServer.swift

```diff
@@ -18,8 +18,10 @@ class WebServer {
     }
 
     func start() -> Bool {
-        if !server.running {
-            try! server.startWithOptions([GCDWebServerOption_Port: 6571, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true])
+        withExtendedLifetime(server) {
```

Comment:
From what I know about withExtendedLifetime, it guarantees that the parameter passed to it will outlive the length of the function that it calls. In this case, the function calls startWithOptions which is synchronous and returns right away so I don't know how extending the lifetime would help in this case. How would this help keep the callbacks alive?